crates/c-api/src/config.rs

```diff
@@ -146,3 +146,18 @@ pub unsafe extern "C" fn wasmtime_config_cache_config_load(
         |_cfg| {},
     )
 }
+
+#[no_mangle]
+pub extern "C" fn wasmtime_config_static_memory_maximum_size_set(c: &mut wasm_config_t, size: u64) {
+    c.config.static_memory_maximum_size(size);
+}
+
+#[no_mangle]
+pub extern "C" fn wasmtime_config_static_memory_guard_size(c: &mut wasm_config_t, size: u64) {
```

Comment:
After testing this from Perl, I think the `_set` suffix was accidentally omitted from this declaration.  Same for `wasmtime_config_dynamic_memory_guard_size` below.